AzureCalling/Controllers/CallViewController.swift

```diff
@@ -520,12 +526,40 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
         meetingInfoViewUpdate()
     }
 
-    @objc func recordingActiveChangeUpdated(_ notification: Notification) {
-        guard let isRecordingActive = callingContext?.isRecordingActive else {
+    private func toggleMeetingCaptureNotification(mode: NoticeBannerType) {
+        guard let isRecordingActive = callingContext?.isRecordingActive,
+              let isTranscriptionActive = callingContext?.isTranscriptionActive else {
             return
         }
-        let notificationText = isRecordingActive ? meetingRecordingActiveText : meetingRecordingStopText
-        messageBannerView.showBannerMessage(notificationText)
+
+        let notificationText: NSAttributedString = {
+            switch (mode, isRecordingActive, isTranscriptionActive) {
+            case (_, true, true):
+                return meetingRecordingAndTranscriptionActiveText
+            case (.recording, true, false):
+                return meetingRecordingActiveText
+            case (.recording, false, true):
+                return meetingRecordingStopTranscriptionActiveText
+            case (.recording, false, false):
+                return meetingRecordingStopText
+            case (.transcription, true, false):
+                return meetingRecordingActiveTranscriptionStopText
+            case (.transcription, false, true):
+                return meetingTranscriptionActiveText
+            case (.transcription, false, false):
+                return meetingTranscriptionStopText
+            }
+        }()
```

Comment:
Creating this closure every time just to return the text does not feel right. I think we can just assign the `notificationText` variable to simplify it.